packages/abstraxion-core/src/AbstraxionAuth.ts

```diff
@@ -285,16 +317,13 @@ export class AbstraxionAuth {
   async login() {
     try {
       const keypair = await this.getLocalKeypair();
-      const granter = this.getGranter();
+      const searchParams = new URLSearchParams(window.location.search);
+      const granter = this.getGranter() || searchParams.get("granter");
 
       if (keypair && granter) {
-        // Already logged in, just update state
+        await this.pollForGrants();
```

Comment:
Do we need to pass in the granter and grantee here?

Reply:
No the core package is slightly different in that it will try to fetch those values internally and if they don't exist, throw